components/InstallHandlerBtn.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <q-btn
+    label="Manage Credentials"
+    :loading="loading"
+    color="primary"
+    v-bind="options"
+    @click="installHandler" />
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2018-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {installHandler} from 'web-credential-handler';
+
+export default {
+  name: 'InstallHandlerButton',
+  props: {
+    // can be used to pass options to the q-btn
+    // @see https://quasar.dev/vue-components/button
+    options: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data() {
+    return {
+      loading: false
+    };
+  },
+  methods: {
+    async installHandler() {
+      try {
+        this.loading = true;
+        // install credential handler
+        await installHandler({url: '/credential-handler'});
+      } catch(e) {
+        // eslint-disable-line no-console
+        console.error('CHAPI register error:', e);
+        this.$q.notify({
+          type: 'negative',
+          message:
+            'Your wallet will not be shown to you as an option for using ' +
+            'credentials on other websites. Please reload the page and ' +
+            'select "Accept" to change this.',
```

Comment:
So this message isn't correct any more -- reloading the page isn't used to call `installHandler` ... it's a button now. We'll need this to say something more appropriate for the new usage.

Reply:
error take was corrected here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/29/commits/b0524dc2ea59ec379cc4dfcefa8fb30c6cd96c4f